content/resources/user/using-api.md

```diff
@@ -72,7 +72,7 @@ print(dl.text) # display response
 
 # Data Upload via API Endpoint Request:
 headers['content-type']='text/tab-separated-values' # add the content-type to header
-u = requests.put('https://gen3.commons.io/api/v0/submission/project-id', data=tsv, headers=headers)
+u = requests.put('https://gen3.commons.io/api/v0/submission/<program>/<project>', data=tsv, headers=headers)
```

Comment:
maybe `<program name>/<project code>` to make it extra clear?

Reply:
done